superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -205,6 +223,16 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
 
       fetchResource(id)
         .then(() => {
+          // Altering 'extra' JSON before returning it to 'extra' text field
+          if (dbFetched?.extra) {
+            // Parse 'extra' field
+            const extraParsed = JSON.parse(dbFetched.extra || defaultExtra);
+            // Remove metadata_cache_timeout and schemas_allowed_for_csv_upload
+            delete extraParsed.metadata_cache_timeout;
+            delete extraParsed.schemas_allowed_for_csv_upload;
```

Comment:
are you saving these values anywhere to display in the form?

Reply:
I'm currently storing them in useState, in the `metaDataCacheTimeout` and `schemasAllowed` variables. This may be moved to useReducer soon though!